blueprints/third-party-solutions/phpipam/cloudsql.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "random_password" "cloudsql_password" {
+  length = 8
+}
+
+# Set up CloudSQL
+module "cloudsql" {
+  source           = "../../../modules/cloudsql-instance"
+  project_id       = module.project.project_id
+  prefix           = var.prefix
+  network          = local.network
+  name             = "${var.prefix}-mysql"
+  region           = var.region
+  database_version = local.cloudsql_conf.database_version
+  tier             = local.cloudsql_conf.tier
+  databases        = [local.cloudsql_conf.db]
+  users = {
+    "${local.cloudsql_conf.user}" = var.cloudsql_password
```

Comment:
if you set the value to null, the module will generate a random password for you

Reply:
Thanks! Got rid of the random password resource indeed!